README.rst

```diff
@@ -7,219 +7,26 @@ Build Status
 ------------
 |build-status|
 
-Features
---------
-* Distributed via PyPI.
-* Easy management of credentials.
-* Easy creation of Google Ads API service clients.
-
 Requirements
 ------------
 * Python 3.7+
-        - **NOTE:** Python 2 support has ceased as of v4.0. See this `blog post`_ for more detail.
-* `pip`_
-
-
-Documentation
--------------
-This README has general information to get you started with the library, but more
-extensive documentation can be found on our `Developer Site`_.
-
-Getting started
----------------
 
 Installation
-############
-
-This library is distributed via PyPI. If you have not already done so, install
-`pip`_, the following command to install this client library:
-
+------------
 .. code-block::
 
   pip install google-ads
```

Comment:
do we even need this line when there is more detailed install info coming to devsite?

Reply:
We decided to leave quick reference info like this here for people who just want to know the installation command. I had initially taken it out but put it back in after talking with @mcloonan